packages/story-editor/src/components/canvas/karma/quickActions.karma.js

```diff
@@ -56,6 +56,60 @@ describe('Quick Actions integration', () => {
 
       expect(fixture.screen.queryByTestId('quick-actions-menu')).toBeNull();
     });
+
+    it('quick menu should not be visible if no quick actions are present', async () => {
+      // when two different elements types are selected, there may not be any quick actions to show
+      // in that case, we shouldn't be rendering the context menu at all
+      const insertElement = await fixture.renderHook(() => useInsertElement());
+      const shapeElement = await fixture.act(() =>
+        insertElement('shape', {
+          backgroundColor: {
+            color: {
+              r: 203,
+              g: 103,
+              b: 103,
+            },
+          },
+          type: 'shape',
+          x: 48,
+          y: 0,
+          width: 148,
+          height: 137,
+          scale: 100,
+          focalX: 50,
+          focalY: 50,
+          mask: {
+            type: 'heart',
+          },
+          id: 'cb89750a-3ffd-4876-8ed9-d715c553e05b',
+          link: null,
+        })
+      );
+
+      await fixture.act(() =>
+        insertElement('text', {
+          font: TEXT_ELEMENT_DEFAULT_FONT,
+          content: 'Hello world!',
+          x: 10,
+          y: 20,
+          width: 400,
+        })
+      );
```

Comment:
💭  Miina asked me a little while back to limit our use of `insertElement` and instead add elements using the library.

Let's replace these with adding an element directly from the library: https://github.com/GoogleForCreators/web-stories-wp/blob/b3a2b97493ecb9f116e7c7015d69ee3b05de8927/packages/story-editor/src/components/floatingMenu/elements/karma/more.karma.js#L43-L94

Reply:
You caught me, I was being lazy.